plugins/civmodcore-paper/src/main/java/vg/civcraft/mc/civmodcore/world/locations/chunkmeta/WorldChunkMetaManager.java

```diff
@@ -228,16 +228,22 @@ private void registerUnloadRunnable() {
 				}
 
 				if (System.currentTimeMillis() - coord.getLastUnloadedTime() > UNLOAD_DELAY) {
+					if (ChunkMetaCommand.chunkMetaLogsEnabled()) {
+						logger.info("[Chunkmeta] Unloading chunk " + coord + " - unloaded: " + coord.getLastUnloadedTime());
+					}
```

Comment:
We shouldn't be logging with guard conditions like this in modern java, we should be using the appropriate log level and a message supplier to prevent resources spent on extra checks and calculations.

This also makes the above command unnecessary.

e.g. 
```java
logger.debug(() -> "[Chunkmeta] Unloading chunk " + coord + " - unloaded: " + coord.getLastUnloadedTime())
```

Reply:
I did some reasearch into this and there's not really any good options for runtime log configuration like this, and since it's temporary I just went for something easy